frontend/src/streamlist/__tests__/StreamsTable.test.jsx

```diff
@@ -19,32 +19,51 @@ import {
 } from "@material-ui/icons";
 import MaterialTable from "material-table";
 import StreamsTable from "../StreamsTable";
-import StreamDetailsButton from "../StreamDetailsButton";
-import StatusIndicator from "../../general/StatusIndicator";
 
 Enzyme.configure({ adapter: new Adapter() });
 jest.spyOn(global.console, "error");
 
 describe("<StreamsTable/> component", () => {
   let wrapper;
+  const dummyStreams = [];
+  const dummyColumns = [
+    {
+      title: "ID",
+      field: "id"
+    },
+    {
+      title: "Date",
+      field: "date"
+    }
+  ];
+
+  beforeEach(() => {
+    wrapper = Enzyme.shallow(
+      <StreamsTable streams={dummyStreams} columns={dummyColumns} />
+    );
+  });
 
   afterEach(() => {
     jest.clearAllMocks();
+    wrapper.unmount();
   });
 
   describe("has the correct components", () => {
-    const dummyStreams = [];
-    beforeEach(() => {
-      wrapper = Enzyme.shallow(<StreamsTable streamDetails={dummyStreams} />);
-    });
     it("contains one Box component", () => {
       expect(wrapper.find(Box)).toHaveLength(1);
     });
     it("contains one TableContainer component", () => {
       expect(wrapper.find(TableContainer)).toHaveLength(1);
     });
-    it("contains one MaterialTable component", () => {
-      expect(wrapper.find(MaterialTable)).toHaveLength(1);
+    describe("contains MaterialTable component", () => {
+      it("that has length of 1 with expected prop values", () => {
+        expect(wrapper.find(MaterialTable)).toHaveLength(1);
+        const tableProps = wrapper.find(MaterialTable).props();
+        expect(tableProps.columns).toBe(wrapper.instance().props.columns);
+        expect(tableProps.data).toBe(wrapper.instance().props.streams);
+        expect(tableProps.options).toEqual(wrapper.instance().getOptions());
+        expect(tableProps.icons).toEqual(wrapper.instance().getIcons());
+      });
```

Comment:
since you're writing these tests. Try to make it in the same format as our new tests
```
it ("Contains 1 <ComponentName/> component with expected props", ()=>{
  const components = wrapper.find(ComponentName);
  expect(components).toHaveLength(1);
  
  const props = components.at(0).props();
  const expected={
    prop: expectedvalue
    ....
  }
  expect (props).toStrictEqual(expected)
})
```